Sources/Roadmap/RoadmapFeatureViewModel.swift

```diff
@@ -23,8 +23,12 @@ final class RoadmapFeatureViewModel: ObservableObject {
     }
 
     @MainActor
-    func getCurrentVotes() async {
-        voteCount = await configuration.voter.fetch(for: feature)
+    func getCurrentVotes(firstLoad: Bool = false) async {
+        if let votes = feature.votes, firstLoad {
```

Comment:
Shouldn't this be:

```suggestion
        if let votes = feature.votes, !firstLoad {
```

If `firstLoad` is set to true, we should "first load the data", so we need to go into the else statement here to fetch for the feature?

Reply:
Intention is to ignore the cache on subsequent loads, using the value from the roadmap JSON if it exists (backwards compatibility)